src/sections/Resources/Resources-grid/filters.js

```diff
@@ -69,138 +72,155 @@ const Navigation = (props) => {
   const meshesFieldValues = meshes.map(mesh => mesh.fieldValue);
 
 
-
   return (
     <ResourceNavigationWrapper>
       <div className="filter">
+        <div onClick={() => {
+          setExpandFilter(!expandFilter); 
+        }} className="filter-menu-icon">
+          {expandFilter ? (<>
+            <IoClose className="filter-icon" />
+            <span>Close Filters</span>
+          </> 
+          )  :
+            (
+              <>
+                <FiFilter className="filter-icon"/>
```

Comment:
![open](https://user-images.githubusercontent.com/85789734/149320919-54fa004c-ade8-4959-b9e6-30f9b5618502.jpg)

Can you enclose this inside a Button? Also, increase the space between the resource card and the button.

Reply:
Align the text to the center of the icon.